autogluon/utils/try_import.py

```diff
@@ -82,3 +82,13 @@ def try_import_faiss():
         raise ImportError(
             "Unable to import dependency faiss"
             "A quick tip is to install via `pip install faiss-cpu`. ")
+
+def try_import_fastai_v1():
+    try:
+        from pkg_resources import parse_version  # pylint: disable=import-outside-toplevel
+        import fastai
+        fastai_version = parse_version(fastai.__version__)
+        assert parse_version('1.0.61') <= fastai_version < parse_version('2.0.0'), 'Currently, we only support 1.0.61<=fastai<2.0.0'
+    except OSError as e:
```

Comment:
When I tried testing this without fastai installed, it threw a `ModuleNotFoundError` instead, so this message was never printed.

Traceback below:
```
    try_import_fastai_v1()
  File "/Users/jonasmue/WorkDocs/AutoGluon/temp/branches/fastainet/autogluon/autogluon/utils/try_import.py", line 89, in try_import_fastai_v1
    import fastai
ModuleNotFoundError: No module named 'fastai'
No module named 'fastai'
```

Reply:
fixed to catch `ModuleNotFoundError`